README.md

```diff
@@ -4,6 +4,17 @@ are now using  the  Web  to  access  vast  amounts  of  information  and  resour
 
 Aim of this Project is to develop a website that assists students and teachers alike to discover online courses, to enroll in courses from different sources, to compare these sources and to track their progress. Students can decide on their own pace and with which session they would like to begin their studies.
 
+# Table of Contents
+
+- [Hacking Online Education](#hacking-online-education)
```

Comment:
I think this is not needed.

Reply:
Maybe take a look at [this](https://github.com/tulikavijay/CBT-therapy#helping-women-cope-with-depression-and-anxiety)